ui/css/treeherder-pinboard.css

```diff
@@ -135,7 +135,15 @@
   color: #98c3da;
 }
 
+.clear-related-bugs-icon {
```

Comment:
Would you see if you can get the right look with the [Bootstrap Utilities](https://getbootstrap.com/docs/4.3/utilities/borders/) classes instead of custom classes?  Like `ml-1` for margin-left.  They are really handy and help keep our custom CSS simpler.  :)

Reply:
removed this custom class for text-danger and ml-2 from bootstrap